alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportGetFindingsAction.kt

```diff
@@ -122,7 +127,8 @@ class TransportGetFindingsSearchAction @Inject constructor(
         client.threadPool().threadContext.stashContext().use {
             scope.launch {
                 try {
-                    val getFindingsResponse = search(searchSourceBuilder)
+                    val indexName = resolveFindingsIndexName(getFindingsRequest)
+                    val getFindingsResponse = search(searchSourceBuilder, indexName)
                     actionListener.onResponse(getFindingsResponse)
                 } catch (t: Exception) {
                     actionListener.onFailure(AlertingException.wrap(t))
```

Comment:
if get monitor fails we would already receive the alerting exception

would we need to do something like
```


if (t is AlertingException) {
    actionListener.onFailure(t)
} else {
      actionListener.onFailure(AlertingException.wrap(t))
 }
```

Reply:
I added another catch for AlertingException